monailabel/transform/post.py

```diff
@@ -37,28 +37,57 @@
 
 
 # TODO:: Move to MONAI ??
-
-
 class LargestCCd(MapTransform):
```

Comment:
for which app/version u r using this class.. ?

This will be deprecated soon..
please use `KeepLargestConnectedComponentd` from monai.transforms instead
https://docs.monai.io/en/latest/transforms.html#keeplargestconnectedcomponentd